src/Symfony/Component/Form/Tests/Extension/Core/Type/RepeatedTypeTest.php

```diff
@@ -105,6 +120,29 @@ public function testSetInvalidSecondOptions()
         ]);
     }
 
+    /**
+     * @param string $configurationKey
+     * @dataProvider notMappedConfigurationKeys
+     */
+    public function testNotMappedInner($configurationKey)
```

Comment:
Then this could be removed.

Reply:
Or better, adapted to have the same behavior as the other.

Reply:
done